public/app/features/variables/query/QueryVariableRefreshSelect.tsx

```diff
@@ -14,14 +15,15 @@ const REFRESH_OPTIONS = [
 ];
 
 export function QueryVariableRefreshSelect({ onChange, refresh }: PropsWithChildren<Props>) {
+  const { width } = useWindowSize();
   const value = useMemo(
     () => REFRESH_OPTIONS.find((o) => o.value === refresh)?.value ?? REFRESH_OPTIONS[0].value,
     [refresh]
   );
 
   return (
     <Field label="Refresh" description="When to update the values of this variable">
-      <RadioButtonGroup options={REFRESH_OPTIONS} onChange={onChange} value={value} />
+      <RadioButtonGroup options={REFRESH_OPTIONS} onChange={onChange} value={value} size={width < 375 ? 'sm' : 'md'} />
```

Comment:
Out of curiosity, why don't we set a width and the labels are automatically truncated? I love this improvement anyways, I like it to get smaller when the screen is too small.

Reply:
I just tried it and it seems it's not possible with the RadioButtonGroup we currently have. I should make some modifications to that core component